integration-tests/bats/sql-local-remote.bats

```diff
@@ -328,8 +328,10 @@ assert_has_key_value() {
 
 @test "sql-local-remote: verify dolt diff behavior with data and schema changes" {
   start_sql_server defaultDB
-
   cd defaultDB
+
+  export DOLT_CLI_PASSWORD=""
```

Comment:
We unset the DOLT_CLI_PASSWORD var in this file, so I'd prefer you just set the `--password` everywhere, OR you can specify no user/pwd (which should work now that my auth changes are in).

Reply:
Cool, dropping `--user` arguments in these tests.